src/components/dashboard/enrolledCardOfCourse/EnrolledCardOfCourse.tsx

```diff
@@ -0,0 +1,61 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import { Box, CardMedia, Typography } from '@mui/material';
+import { EnrolledCoursesData } from '../../../types/components/componentType';
+import './_enrolledCard.scss';
```

Comment:
import SCSS file like module component ?

Reply:
Classnames are not used in tsx